### ISSUE TITLE
[WebGPU] BatchedMesh render nothing on r166

### ISSUE THREAD

**Issue by Cerberus** [mrdoob/three.js#28757]
### Description

MatchedMesh render nothing with warning..

```
AttributeNode.js:92 AttributeNode: Vertex attribute "batchId" not found on geometry.
```

### Reproduction steps

Goto the link below.

### Code

-

### Live example

https://threejs.org/examples/?q=webgpu#webgpu_mesh_batch

### Screenshots

![preview](https://github.com/mrdoob/three.js/assets/9087409/1e1d47e1-6eb8-4286-835f-cdc686f10c3c)

### Version

r166

### Device

Desktop

### Browser

Chrome

### OS

MacOS

**Comment by Cerberus** [comment]
Chrome Version 126.0.6478.127 (Official Build) (x86_64)

**Comment by Mugen87** [comment]
Should be fixed via #28753.
